.github/workflows/pr.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on:
+  pull_request: {}
+
+jobs:
+  main:
+    name: Build, Validate and Deploy
```

Comment:
We only run the release checker here, right? Is this name accurate?

Reply:
Nice catch, I will update the name.